frb_dart/lib/src/typed_data/io.dart

```diff
@@ -69,13 +69,14 @@ class Uint64List extends _TypedList<BigInt> {
   Uint64List.sublistView($data.TypedData data, [int start = 0, int? end])
       : inner = $data.Uint64List.sublistView(data, start, end);
 
-  static final _maxI64 = BigInt.from(0x7FFFFFFFFFFFFFFF);
-  static const _minI64 = 0x8000000000000000;
+  static final _maxi64b = BigInt.from(0x7FFFFFFFFFFFFFFF);
+  static const _mini64 = 0x8000000000000000;
```

Comment:
uber-nit: The old "minI64" looks better to me, since "mini64" looks like "mini 64" instead of "min i64"